### PR TITLE
new VTK point probe

### PR DESCRIPTION
**Pull request by vlukes**

### CODE REVIEW

**Comment by rc** [comment]
Thanks! Do you plan to use all the various probes in an example? So that the code is tested somewhere...

File: sfepy/postprocess/probes_vtk.py [hunk 1]
```diff
@@ -110,6 +110,33 @@ def add_line_probe(self, name, p0, p1, n_point):
...
+    def add_points_probe(self, name, coors):
+        """
+        Create the point probe - VTK object.
+
+        Parameters
+        ----------
+        name : str
+            The probe name.
+        coors : array
+            The cooedinates of the probe points.
```
Comment (rc) [inline sfepy/postprocess/probes_vtk.py]: cooedinates -> coordinates

**Comment by rc** [comment]
@vlukes did you get the notification about my comment? This PR could be merged, after you fix the typo :)

**Comment by vlukes** [comment]
No, I did not get the notification about your comment.

Now, the typo is fixed and the branch is rebased to the current master.

**Comment by rc** [comment]
OK, LGTM, thanks! Adding later an example using the different probes would be nice. Merging now.

**Pull request merged**